### PR TITLE
Stop using travis-ci

### PR DESCRIPTION
**Pull request by coolo**
We test code commits on circleci and merges in the OBS
packages. Travis-CI is pretty much unmaintained (e.g. the
gem caches are broken atm and noone cares)

So remove one out of 3 CI systems

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -145,16 +145,10 @@ namespace :docker do
 end
 
 def environment_vars(with_coverage = true)
-  environment = travis_environment_variables
+  environment = []
   environment.concat(['-e', 'DO_COVERAGE=1']) if with_coverage && ENV['CIRCLECI']
   environment.concat(['-e', 'EAGER_LOAD=1'])
   environment.concat(['-e', "TEST_SUITE='#{ENV['TEST_SUITE']}'"])
   environment
 end
```
Comment (Ana06) [inline Rakefile]: Rubocop doesn-t like this line :unamused:

Comment (coolo) [reply Rakefile]: no need to 'request changes' if the tests are failing

Comment (Ana06) [reply Rakefile]: nut I was so nice to tell you even without not need for it :roll_eyes:

Comment (coolo) [reply Rakefile]: I'm not saying you should not complain - but the 'request changes' tends to stick in PRs :)

**Pull request merged**